packages/store-indexer/src/sqlite/createIndexer.ts

```diff
@@ -0,0 +1,52 @@
+import { Chain, PublicClient, Transport } from "viem";
+import {
+  createBlockStream,
+  isNonPendingBlock,
+  blockRangeToLogs,
+  groupLogsByBlockNumber,
+} from "@latticexyz/block-logs-stream";
+import { concatMap, filter, from, map, mergeMap, tap } from "rxjs";
+import { storeEventsAbi } from "@latticexyz/store";
+import { blockLogsToStorage } from "@latticexyz/store-sync";
+import { sqliteStorage } from "@latticexyz/store-sync/sqlite";
+import { BaseSQLiteDatabase } from "drizzle-orm/sqlite-core";
+import { debug } from "../debug";
+
+type CreateIndexerOptions = {
+  database: BaseSQLiteDatabase<"sync", any>;
+  publicClient: PublicClient<Transport, Chain>;
+  startBlock: bigint;
+  maxBlockRange: bigint;
+};
+
+export function createIndexer({ database, publicClient, startBlock, maxBlockRange }: CreateIndexerOptions): void {
```

Comment:
could we add some tsdoc style code documentation to this function? (ideally for all functions exported by the package)